.github/workflows/sync-assets-to-plugin-repo.yml

```
@@ -0,0 +1,44 @@
+name: Sync Gutenberg plugin assets to WordPress.org plugin repo
+
+on:
+    push:
+        branches:
+            - trunk
+        paths:
+            - assets
```

I think this needs a pattern based on docs and examples [here](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-including-paths) and [especially here](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet):

```suggestion
            - assets/**
```

---

Addressed in https://github.com/WordPress/gutenberg/pull/68052/commits/bd07fed8c2acb8337b74b6054163f6e3babc1951.